packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -36,11 +36,24 @@ export interface ILoadTest {
 	run(config: IRunConfig, reset: boolean): Promise<boolean>;
 	detached(config: Omit<IRunConfig, "runId" | "profileName">): Promise<LoadTestDataStoreModel>;
 	getRuntime(): Promise<IFluidDataStoreRuntime>;
+	waitLoaded(): Promise<void>;
+	/**
+	 * Generate ops.
+	 * @param n - number of ops to generate
+	 */
+	generateChanges(n: number);
+	/**
+	 * Verify that the expected number of stashed ops have been applied to the counter
+	 * @param expected - expected number of stashed ops
+	 * @returns true if the counter matches the expected value
+	 */
+	verify(expected: number): boolean;
```

Comment:
i don't think we should add this. this test isn't meant to verify specific features.